blockchains/smartchain/tokenlist.json

```diff
@@ -157,6 +175,84 @@
       "logoURI":"https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/smartchain/assets/0x2170Ed0880ac9A755fd29B2688956BD959F933F8/logo.png",
       "pairs":[
 
+      ]
+    },
+    {
+      "chainId":56,
+      "asset":"c20000714_t0x55d398326f99059fF775485246999027B3197955",
```

Comment:
It seems to be BUSD token and not Tether: https://bscscan.com/address/0x55d398326f99059fF775485246999027B3197955

Reply:
They have changed the name of the token recently. But contract shows this is BEP20 USDT.
![image](https://user-images.githubusercontent.com/35627271/100962997-791cdf80-3560-11eb-8112-1fcd96539a14.png)

Here is contract for BUSD: https://bscscan.com/token/0xe9e7cea3dedca5984780bafc599bd69add087d56